src/api/routes/stx-supply.ts

```diff
@@ -70,7 +74,11 @@ export const StxSupplyRoutes: FastifyPluginAsync<
                   'String quoted decimal number of the percentage of STX that have unlocked',
               }),
               total_stx: Type.String({
-                description: 'String quoted decimal number of the total possible number of STX',
+                description: 'String quoted decimal number of the total circulating number of STX',
+              }),
+              total_stx_year_2050: Type.String({
+                description:
+                  'The Stacks cryptocurrency has a predefined future supply that reaches approx 1,818M STX by year 2050',
```

Comment:
I'd say: "String quoted decimal number of total circulating STX supply in year 2050. STX supply grows approx 0.3% annually thereafter in perpetuity."

Reply:
Updated